electron/launcher.ts

```diff
@@ -484,11 +485,22 @@ async function runLegendaryOrGogdlCommand(
     child.on('close', (code, signal) => {
       errorHandler({
         error: { stderr: stderr.join(), stdout: stdout.join() },
-        logPath: options?.logFile
+        logPath: options?.logFile,
+        runner: runner.name
       })
+
+      if (stderr.join().includes('ERROR')) {
+        rej(stderr.join())
+      }
+
+      if (stderr.join().includes('MemoryError:')) {
+        rej('MemoryError:')
+      }
```

Comment:
It might be better if we'd just reject the promise here and let `LegendaryGame.install` handle figuring out what the exact error was